core/services/keystore/keys/ocr2key/terra_keyring.go

```diff
@@ -57,7 +57,14 @@ func (ok *terraKeyring) Sign(reportCtx ocrtypes.ReportContext, report ocrtypes.R
 	return utils.ConcatBytes(ok.PublicKey(), signedMsg), nil
 }
 
+// Note signature is prefixed with the public key.
 func (ok *terraKeyring) Verify(publicKey ocrtypes.OnchainPublicKey, reportCtx ocrtypes.ReportContext, report ocrtypes.Report, signature []byte) bool {
+	if len(signature) != ok.MaxSignatureLength() {
```

Comment:
yeah it can't be, we implement the Sign above. Not 100% sure why its named that way in libocr, but my guess is to support cases keyrings that handle both compressed and uncompressed keys

Reply:
Perhaps `ok` should be `tk`?